src/main/java/org/apache/commons/lang3/builder/DiffResult.java

```diff
@@ -88,6 +88,22 @@
         }
     }
 
+    /**
+     * <p>Return the object the right object has been compared to</p>
```

Comment:
- You do not need to use p tags for the first sentence of a Javadoc comment IIRC. 
- A sentence should end in a period.
- Use "Gets ..." instead of "Return ..."

Reply:
thanks @garydgregory 
which version should I put in the `@since` tag ?

Reply:
Also maybe I could call the getters `getLeft` and `getRight` ... seems more readable, what do you think ?

Reply:
@nicolasbd
Yes, please use `getLeft` and `getRight`, much better!

Reply:
@garydgregory ok cool, but what about the `@since` tag ? which number should I set it to ?

Reply:
3.10

Reply:
Ok @garydgregory , made the changes except for "Returns" and p tags for consistency with other javadocs comments